sw/airborne/modules/guidance/gvf/gvf.c

```diff
@@ -178,37 +180,19 @@ void gvf_control_2D(float ke, float kn, float e,
   float md_dot_y = -md_x * md_dot_const;
 
 
-  #ifdef ROTORCRAFT_FIRMWARE
+  #if defined(ROTORCRAFT_FIRMWARE)
 
-  // Set nav for command
+  nav.setpoint_mode = NAV_SETPOINT_MODE_SPEED;
 
-  // Use parameter kn as the speed command
-  nav.speed.x = md_x * kn;
-  nav.speed.y = md_y * kn;
+  // Speed-based control, acceleration based control not implemented yet
+  nav.speed.x = gvf_control.speed * md_x;
+  nav.speed.y = gvf_control.speed * md_y;
 
-
-  // Acceleration induced by the field with speed set to kn (!WIP!)
```

Comment:
The control in acceleration is possible with some controllers (INDI) but not all (PID). Still I think the two options should be kept, either selected with a compilation flag (something like `GVF_ROTORCRAFT_USE_ACCEL`, or any better name), or a dynamic selection at runtime. Unless there is a good reason not to keep the accel based command ?

Reply:
We decided to use the speed control due to simplicity, however, I can include the older implementation with the flag as you said. Apart from that and AFAIK, the older implementation doesn't work, does it?

Reply:
It was supposed to work, but not as good as expected. We will test and fix it later.